cluster-autoscaler/core/equivalence_groups_test.go

```diff
@@ -56,10 +66,16 @@ func TestGroupSchedulablePodsForNode(t *testing.T) {
 	p3_1.OwnerReferences = GenerateOwnerReferences(rc2.Name, "ReplicationController", "extensions/v1beta1", rc2.UID)
 	p3_2 := BuildTestPod("p3_2", 100, 200000)
 	p3_2.OwnerReferences = GenerateOwnerReferences(rc2.Name, "ReplicationController", "extensions/v1beta1", rc2.UID)
-	unschedulablePods := []*apiv1.Pod{p1, p2_1, p2_2, p3_1, p3_2}
+	p4_1 := BuildTestPod("p4_1", 100, 200000)
```

Comment:
Can you add a case for non-projected volume to verify they still end up in different equivalence groups?

Reply:
good call - updated!